components/Movies.js

```diff
@@ -0,0 +1,183 @@
+import React, {useState, useEffect} from 'react';
+// importing core componenets from react native
+import {
+	View, 
+	Text, 
+	ActivityIndicator, 
+	Image, 
+	Button, 
+	TouchableOpacity, 
+	ScrollView, 
+	StatusBar,
+	SafeAreaView,
+	StyleSheet, 
+	Share
+} from 'react-native';
+import styled from 'styled-components/native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import { FETCH_URL } from '../utils/urls';
+
+const Movies = () => {
+
+	const [movieList, setMovieList] = useState([]);
+	const [loading, setLoading] = useState(false);
+
+	// random movie from movieList array, returns only one item
+	const randomMovie = movieList[Math.floor(Math.random() * 	movieList.length)];
+
+	useEffect(() => {
+		generateMovie();
+	}, []);
+
+	const generateMovie = () => {
+		fetch(FETCH_URL)
+			.then((res) => res.json())
+			.then((json) => {
+				setMovieList(json.results);
+				//setLoading(true)
+			})
+			.finally(() => setLoading(false))
+	};
+
```

Comment:
By commenting out the loading in line 38, there's no use of the ActivityIndicator at all in this component. If I remember right from your demo, there was some interference with your other code, if you put the setLoading(true) in line 34 where it belongs. So maybe delete it altogether, if you don't have use for it or try to fix the issue.

Reply:
Yes! Thanks for the note! :)